source/user-manual/capabilities/vulnerability-detection/how_it_works.rst

```diff
@@ -17,31 +17,42 @@ The global vulnerability database is created automatically, currently pulling da
 
 This database can be configured to be updated periodically, ensuring that the solution will check for the very latest CVEs.
 
-Once the global vulnerability database (with the CVEs) is created, the detection process looks for vulnerable packages in the inventory databases (unique per agent). Alerts are generated when a CVE (Common Vulnerabilities and Exposures) affects a package that is known to be installed in one of the monitored servers. A package is labeled as vulnerable when its version is contained within the affected range of a CVE.
+Once the global vulnerability database (with the CVEs) is created, the detection process looks for vulnerable packages in the inventory databases (unique per agent). Alerts are generated when a CVE (Common Vulnerabilities and Exposures) affects a package that is known to be installed in one of the monitored hosts. A package is labeled as vulnerable when its version is contained within the affected range of a CVE.
 The results are presented as alerts and also stored in a database. So you can check the last scan alerts, or query every single agent's vulnerable software DB.
 
 .. _vuln_det_scan_types:
 
 Scan types
 ^^^^^^^^^^
 
-The Vulnerability Detector module can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
-In any of these cases, the packages and the operating system that were already scanned will be re-scanned if the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires.
-This improves the performance and avoids repeated alerts during a configurable time. We have then three different types of scan:
+The `Vulnerability Detector` module performs a scan when it is enabled generating a vulnerabilities inventory of packages and operating system installed in the managed agents.
+It also can run a scan on startup (:ref:`run_on_start <vuln_det_run_on_start>`) and every certain period of time (:ref:`interval <vuln_det_interval>`).
+In any of these cases, the packages and the operating system that were already scanned will not be re-scanned, only the new ones will do and alerts are going to be generated.
+If the database of vulnerabilities receives new CVEs information and :ref:`min_full_scan_interval <vuln_det_min_full_scan_interval>` expires all packages as well as the operating system will be re-scanned.
```

Comment:
Considering Diego´s suggestion in line 29, we shouldn´t mention _**In any of these cases**_. It will be clearer to not differentiate the first scan done on startup here as can be confused with the baseline functionality